doc/source/security/using-keyring-plugin.rst

```diff
@@ -0,0 +1,120 @@
+.. _using-keyring-plugin:
+
+=======================================================
+Using the Keyring Plugin
+=======================================================
+
+|Percona Server| provides the following plugins:
+
+*  `keyring_file` stores the keyring data locally
+
+* `keyring_encrypted_file` stores the keyring data in an encrypted file located on the server.
```

Comment:
we do not provide keyring_encrypted_file

Reply:
changed to Percona Server may use the following:

Reply:
dropped the keyring_encrypted_file.